lib/Cake/Utility/Inflector.php

```diff
@@ -495,7 +495,12 @@ public static function underscore($camelCasedWord) {
  */
 	public static function humanize($lowerCaseAndUnderscoredWord) {
 		if (!($result = self::_cache(__FUNCTION__, $lowerCaseAndUnderscoredWord))) {
-			$result = ucwords(str_replace('_', ' ', $lowerCaseAndUnderscoredWord));
+			$result = str_replace('_', ' ', $lowerCaseAndUnderscoredWord);
+			if (function_exists('mb_convert_case') && Multibyte::checkMultibyte($result)) {
```

Comment:
I would just always use mb_convert_case if it exists. There isn't much point in checking the string.